scripts/install.sh

```diff
@@ -78,4 +78,4 @@ if [[ -z $NO_INSTALL ]]; then
 fi
 
 version_check
-
+tilt verify-install
```

Comment:
can you also add this 
- here: https://github.com/tilt-dev/tilt/blob/master/scripts/install.ps1#L49 (for windows)
- here: https://github.com/tilt-dev/tilt/blob/master/.goreleaser.yml#L49 (for homebrew)